ddpui/management/commands/set_uuids_in_orgtask_deployment.py

```diff
@@ -0,0 +1,206 @@
+from ddpui.models.org import OrgDataFlowv1
+from ddpui.models.tasks import OrgTask, DataflowOrgTask
+from ddpui.ddpprefect import prefect_service
+from ddpui.utils.deleteorg import delete_one_org, display_org
+from ddpui.ddpprefect.schema import PrefectDataFlowUpdateSchema3
+import logging
+
+load_dotenv()
+
+logger = logging.getLogger("migration")
+
+
+class Command(BaseCommand):
+    """
+    This script deletes an org and all associated entities
+    Not only in the Django database, but also in Airbyte and in Prefect
+    """
+
+    help = "Deletes an organization"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--slug",
+            required=False,
+            default="all",
+            help="By default it will run for all organization. To run for an org; pass the slug here",
+        )
+
+    def set_uuids_to_orgtasks(self, org: Org):
+        """Sets uuids to those orgtasks that have currently a null"""
+        logger.info(f"setting uuids for org {org.slug}")
+        for org_task in OrgTask.objects.filter(org=org).all():
+            if org_task.uuid is None:
+                org_task.uuid = uuid.uuid4()
+                org_task.save()
+                logger.info(f"updated uuid for org_task {org_task.id}")
+
+    def update_deployment_params_with_orgtask_uuid(self, org: Org):
+        """
+        Updates deployment params with orgtask uuid
+        1) Fetched all dataflows for the org
+        2) For each dataflow, fetched the deployment from prefect and its params
+        3) Use slug to match deployment param task with orgtask
+        4) Push the update task config in deployment params with the uuid of orgtask
+        5) Update the deployment with updated params in prefect
+        """
+        logger.info(
+            f"updating deployment params with uuids of orgtask for org {org.slug}"
+        )
+        for dataflow in OrgDataFlowv1.objects.filter(org=org).all():
+            logger.info(
+                f"processing dataflow with deployment_id {dataflow.deployment_id}"
+            )
+            # fetch deployment params
+            deployment = None
+            try:
+                deployment = prefect_service.get_deployment(dataflow.deployment_id)
+            except Exception as error:
+                logger.info(
+                    f"Something went wrong in fetching the deployment with id '{dataflow.deployment_id}'"
+                )
+                logger.exception(error)
+                logger.info("skipping to next loop")
+                continue
+
+            # add uuid to each task config in params
+            params = deployment["parameters"]
+            if "config" in params and "tasks" in params["config"]:
+                for task_config in params["config"]["tasks"]:
+                    if "slug" in task_config:
+                        org_task = OrgTask.objects.filter(
+                            org=org, task__slug=task_config["slug"]
```

Comment:
over here... for the dbt orgtasks do you want to filter on `parameters__in=["", "{}"]` just in case some client has created a custom orgtask